From cff95045d57d78e263d17dfa8eaf737b9e3aee56 Mon Sep 17 00:00:00 2001
From: Zaikuo Wang <zaikuo.wang@intel.com>
Date: Thu, 2 Apr 2015 17:17:00 +0800
Subject: [PATCH] atomisp2:add one css reset event for HAL

As dual camera run and close one, the isp would stop and reset,
then exp_id would be set to 0 for another camera;
this would cause HAL not to find correct event
so we add this reset event to notify HAL this point

Change-Id: I325e7aa8b2a1ad7d93cfc17ecfb08d95e4c8c96c
Tracked-On: #JIMINAN-25665
Signed-off-by: Zaikuo Wang <zaikuo.wang@intel.com>
---
 include/linux/atomisp.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index d20de5a..91000a1 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1323,7 +1323,7 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define V4L2_EVENT_ATOMISP_RAW_BUFFERS_ALLOC_DONE   (V4L2_EVENT_PRIVATE_START + 3)
 #define V4L2_EVENT_ATOMISP_ACC_COMPLETE     (V4L2_EVENT_PRIVATE_START + 4)
 #define V4L2_EVENT_ATOMISP_PAUSE_BUFFER	    (V4L2_EVENT_PRIVATE_START + 5)
-
+#define V4L2_EVENT_ATOMISP_CSS_RESET	    (V4L2_EVENT_PRIVATE_START + 6)
 /* Nonstandard color effects for V4L2_CID_COLORFX */
 enum {
 	V4L2_COLORFX_SKIN_WHITEN_LOW = 1001,
-- 
1.9.1

